Restructure Docs (#18)
CodeMC/Documentation: docs/services/ci/setups/gradle-job.md

///

1. Go to `https://ci.codemc.io/job/{user|organization}/newJob`
2. Enter a name for your new project and select **Freestyle project**. Press the **OK** button at the bottom.
3. Jenkins should now create a new Job for you and redirect you to its configuration page once done.

## Configuration
We recommend to set the following settings for your project:

### Source Code Management

1. In the **Source Code Management** Section, select **Git**
2. Press **Add Repository**.
3. Enter your repository's URL in the **Repository URL** field.
4. If you only want specific branches to trigger a Build can you add a **Branch Specifier** by pressing **Add Brnach**, if there isn't already one set.

### Build Triggers
/// info | This is optional
You can setup automatic Builds for GitHub, GitLab, or any other supported Repository Host.  
Tutorials can be found in the [Integration Pages](../integrations/index.md) of this documenation.
///

1. In the **Build Triggers** Section, check **Poll SCM**
2. Put `*/10 * * * *` in the large text field to set CodeMC to check your remote repository every 10 minutes.

### Build Environment
/// info | This is optional
///

> We recommend enabling **Add timestamps to the Console Output**

Reviewer: ```suggestion
We recommend enabling **Add timestamps to the Console Output**.
```

### Build
/// warning | Important
In order to build a project with JDK 8, you need to use a [workaround](../../../faq/build-jdk-8-project.md).
///

1. In the **Build** Section, press the **Add build step** button and select **Invoke Gradle script** from the dropdown menu.
2. Press the **Use Gradle Wrapper** Radio Button.
4. Make sure **Make gradlew executable** is selected.
5. Add your tasks in the **Tasks** field. Those are usually `clean build`, `clean shadowJar` or similar.
    - You can deploy your artifacts to the CodeMC Nexus. A tutorial can be found [here](../../nexus/deploy.md).

### Post-build Actions

1. Press the **Add post-build action** button and select **Archive the artifacts** from the dropdown menu.
2. Assuming a default build-directory, set `build/libs/*.jar` (`**/build/libs/*.jar` for multi-module projects) as the value of the **Files to archive** field.

## Final Steps

> 1.  Press the **Save** Button to save your changes and be redirected to your Project's Job page.

Reviewer: ```suggestion
1.  Press the **Save** Button to save your changes, redirecting you to your Project's Job page.
```

Author: Sounds a bit odd to me... Maybe ", which redirects you to your Project's Job page after."?

Reviewer: I think future tense is better in this situation, so I changed it to `which will redirect` instead of `which redirects`.
